client/src/components/utils/Header/Menu/style.css

```diff
@@ -7,6 +7,8 @@
     background: #231f20;
     color: #fff;
     padding-top: 30px;
+    position: fixed;
```

Comment:
@ahmedisam99 Hey! Just passing on something to watch out for: iOS devices sometimes are very buggy with `position: fixed` elements (if you scroll, sometimes you see headers jumping down and then correcting themselves to be back up in their fixed location).  So if that happens in your app, you might have to find a way to change this to position: absolute.